### PR TITLE
cfp: update deadline date

### PR DESCRIPTION
**Pull request by cmaureir**
Adding 7 more days for the cfp extension.

### CODE REVIEW

File: data/pages-content/cfp.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ subtitle: EuroPython Call for Proposals
 **EuroPython 2024 Call for Proposals (CFP) is Open.**
 
 It will be running between
-**Wednesday, 14th February 2024** and **Wednesday, 6th March 2024**
+**Wednesday, 14th February 2024** and **Wednesday, <s>6th</s> 13th, March 2024**
```
Comment (cmaureir) [inline data/pages-content/cfp.md:11]: I hope the tag `<s>` works.

Comment (cmaureir) [reply data/pages-content/cfp.md:11]: Screenshot 2024-03-07 at 23-29-22 CFP July 8th-14th 2024 Prague Czech Republic   Remote

It does, but it's a bit unclear.

Comment (cmaureir) [inline data/pages-content/cfp.md:11]:
```suggestion
**Wednesday, 14th February 2024** and **Wednesday,** <s>6th</s> **13th, March 2024**
```

**Comment by cmaureir** [comment]
Now's better IMHO
Screenshot 2024-03-07 at 23-32-34 CFP July 8th-14th 2024 Prague Czech Republic   Remote

**Review by artcz — APPROVED** [review]

**Pull request merged**